sway-fmt-v2/src/fmt.rs

```diff
@@ -359,27 +359,6 @@ storage {
         assert_eq!(correct_sway_code, formatted_sway_code)
     }
     #[test]
-    fn test_storage_single_line() {
```

Comment:
why was this test removed?

Reply:
I was under the impression we didn't want single line formatting for `storage`, but if that's not the case I can absolutely replace it